rr_iarrc/src/laplacian_line_detection/laplacian_line_detector.cpp

```diff
@@ -0,0 +1,161 @@
+#include <ros/ros.h>
+#include <ros/publisher.h>
+#include <cv_bridge/cv_bridge.h>
+#include <sensor_msgs/Image.h>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/opencv.hpp>
+#include <opencv2/imgcodecs.hpp>
+#include <opencv2/opencv.hpp>
+#include <iostream>
+#include <stdlib.h>
+#include <stdio.h>
+
+using namespace cv;
+
+cv_bridge::CvImagePtr cv_ptr;
+ros::Publisher pub, pub1, pub2, pub3;
+
+cv::Mat kernel(int, int);
+cv::Mat fillColorLines(cv::Mat, cv::Mat);
+void cutEnvironment(cv::Mat);
+cv::Mat cutSmall(cv::Mat, int);
+void publishMessage(ros::Publisher, Mat, std::string);
+Mat overlayBinaryGreen(Mat, Mat);
```

Comment:
I like that you're using forward declarations. Helps give people an overview of what to expect in this file